src/components/organisms/OcModal/OcModal.vue

```diff
@@ -34,6 +34,9 @@
             @keydown.enter="confirm"
           />
           <p v-else key="modal-message" class="oc-modal-body-message" v-text="message" />
+          <div v-if="checkbox" class="oc-modal-body-actions oc-flex oc-flex-left">
+            <oc-checkbox size="medium" :value="checkboxValue" :label="checkboxLabel" :aria-label="checkboxLabel"/>
```

Comment:
You're never setting the `checkboxValue`. Needs two way data binding via `v-model="checkboxValue`.
I guess we also want to allow setting an initial value for the checkbox.

Reply:
yes true